io.github.fheroes2.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.fheroes2</id>
```

Comment:
please review the app-id documentation. This app-id doesn't fit the requirements.

https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
I've changed it to "io.github.ihhub.Fheroes2". Is this one okay?
Do I need another pull request with "io.github.ihhub.Fheroes2" as branch-name?

Reply:
the branch name is completely irrelevant